eng/pipelines/spec-gen-sdk-java.yml

```diff
@@ -0,0 +1,51 @@
+parameters:
+  - name: SpecRepoBranch
+    type: string
+    default: 'main'
+  - name: SpecRepoCommit
+    type: string
+    default: 'HEAD'
+  - name: SdkRepoBranch
+    type: string
+    default: 'main'
+  - name: ConfigType
+    type: string
+    values:
+      - 'TypeSpec'
+      - 'OpenAPI'
+    default: 'TypeSpec'
+    displayName: 'Configuration Type'
+  - name: TspConfigPath
+    type: string
+    default: 'specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml'
+    displayName: 'Path to TypeSpec Configuration file'
+  - name: ReadmePath
+    type: string
+    default: 'specification/contosowidgetmanager/data-plane/readme.md'
+    displayName: 'Path to OpenAPI Configuration'
+  - name: ApiVersion
+    type: string
+    default: 'not-specified'
+    displayName: 'API Version of Spec to be used for SDK Generation'
+  - name: SkipPullRequestCreation
+    type: boolean
+    default: false
```

Comment:
Do we want the default to be create a PR?

Reply:
Yes.